Podfile.lock

```diff
@@ -401,7 +401,7 @@ PODS:
     - WordPressKit (~> 4.18-beta)
     - WordPressShared (~> 1.12-beta)
     - WordPressUI (~> 1.7-beta)
-  - WordPressKit (4.32.0-beta.3):
+  - WordPressKit (4.32.0-beta.4):
```

Comment:
I think this change can be reverted since https://github.com/wordpress-mobile/WordPress-iOS/pull/16342 will be updating this.

Reply:
Since this was branched off of that branch this change should be resolved as soon as that other one merges.

Reply:
Makes sense, thanks!